modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/application/openapi/contributor/test/APIApplicationOpenAPIContributorTest.java

```diff
@@ -0,0 +1,574 @@
+							).put(
+								"objectFieldERC", "DATE_TIME_FIELD"
+							),
+							JSONUtil.put(
+								"description", "decimalProperty description"
+							).put(
+								"name", "decimalProperty"
+							).put(
+								"objectFieldERC", "DECIMAL_FIELD"
+							),
+							JSONUtil.put(
+								"description", "integerProperty description"
+							).put(
+								"name", "integerProperty"
+							).put(
+								"objectFieldERC", "INTEGER_FIELD"
+							),
+							JSONUtil.put(
+								"description", "longIntegerProperty description"
+							).put(
+								"name", "longIntegerProperty"
+							).put(
+								"objectFieldERC", "LONG_INTEGER_FIELD"
+							),
+							JSONUtil.put(
+								"description", "longTextProperty description"
+							).put(
+								"name", "longTextProperty"
+							).put(
+								"objectFieldERC", "LONG_TEXT_FIELD"
+							),
+							JSONUtil.put(
+								"description",
+								"multiselectPicklistProperty description"
+							).put(
+								"name", "multiselectPicklistProperty"
+							).put(
+								"objectFieldERC", "MULTISELECT_PICKLIST_FIELD"
+							),
+							JSONUtil.put(
+								"description", "picklistProperty description"
+							).put(
+								"name", "picklistProperty"
+							).put(
+								"objectFieldERC", "PICKLIST_FIELD"
+							),
+							JSONUtil.put(
+								"description",
+								"precisionDecimalProperty description"
+							).put(
+								"name", "precisionDecimalProperty"
+							).put(
+								"objectFieldERC", "PRECISION_DECIMAL_FIELD"
+							),
+							JSONUtil.put(
+								"description", "richTextProperty description"
+							).put(
+								"name", "richTextProperty"
+							).put(
+								"objectFieldERC", "RICH_TEXT_FIELD"
+							),
+							JSONUtil.put(
+								"description", "textProperty description"
+							).put(
+								"name", "textProperty"
+							).put(
+								"objectFieldERC", "TEXT_FIELD"
+							))
+					).put(
+						"description", "description"
+					).put(
+						"externalReferenceCode", "SCHEMA"
+					).put(
+						"mainObjectDefinitionERC",
+						_objectDefinition1.getExternalReferenceCode()
+					).put(
+						"name", "SchemaName"
+					))
+			).put(
+				"applicationStatus", "published"
+			).put(
+				"baseURL", "test"
+			).put(
+				"externalReferenceCode", "APPLICATION"
+			).put(
+				"title", "title"
+			).toString(),
+			"headless-builder/applications", Http.Method.POST);
+		HTTPTestUtil.invoke(
+			null,
+			"headless-builder/schemas/by-external-reference-code/SCHEMA" +
+				"/requestAPISchemaToAPIEndpoints/ENDPOINT",
+			Http.Method.PUT);
+		HTTPTestUtil.invoke(
+			null,
+			"headless-builder/schemas/by-external-reference-code/SCHEMA" +
+				"/responseAPISchemaToAPIEndpoints/ENDPOINT",
+			Http.Method.PUT);
+
+		OpenAPI openAPI = new OpenAPI();
+
+		_apiApplicationOpenAPIContributor.contribute(
+			openAPI,
+			new OpenAPIContext() {
+				{
+					setPath("/o/test/");
+				}
+			});
+
+		JSONUtil.equals(
```

Comment:
Maybe I am missing something but shouldn't it have an assert? I don't see any `assert` on the test 😅

Reply:
😂 you are totally right